core/src/env_platform/db_handler.bal

```diff
@@ -0,0 +1,79 @@
+import ballerina/config as conf;
+import ballerina/mongodb;
+
+// Mongodb configurations.
+mongodb:ClientConfig mongoConfig = {
+    host: conf:getAsString("DB_HOST"),
+    username: conf:getAsString("DB_USER_NAME"),
+    password: conf:getAsString("DB_PASSWORD"),
+    options: {sslEnabled: false, serverSelectionTimeout: 500}
+};
+mongodb:Client mongoClient = check new (mongoConfig);
+mongodb:Database mongoDatabase = check mongoClient->getDatabase("EnvironmentPlatform");
+mongodb:Collection applicationCollection = check mongoDatabase->getCollection("applications");
+
+# The `postApplicationToDB` function will post the application to the applications collection in the database
+# 
+# + form - The TreeRemocalForm Type record is accepted.
+# 
+# + return - This function will return null if application is added to the database or else return mongodb:Database error
+function postApplicationToDB(TreeRemovalForm form) returns error? {
+
+    json[] locations = [];
+    foreach Location location in form.area {
+        locations.push(<json>{"Latitude": location.latitude, "longitude": location.longitude});
+    }
+    json[] treeInformation = [];
+    foreach TreeInformation treeInfo in form.treeInformation {
+        json[] logDetails = [];
+        foreach var item in treeInfo.logDetails {
+            logDetails.push(<json>{"minGirth": item.minGirth, "maxGirth": item.maxGirth, "height": item.height});
+        }
+        treeInformation.push(<json>{
+            "species": treeInfo.species,
+            "treeNumber": treeInfo.treeNumber,
+            "heightType": treeInfo.heightType,
+            "height": treeInfo.height,
+            "girth": treeInfo.girth,
+            "logDetails": logDetails
+        });
+    }
+    map<json> application = {
+        "applicationId": "tcf-20200513",
```

Comment:
Is application id going to be static?

Reply:
No machan this is only for the code and we have to way of generating the ID.

Reply:
Okay, for now let's have that

Reply:
As an example for the ID I have used <form-type>-<date> which is not good.